### PR TITLE
Update playlist component test coverage

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update playlist component test coverage

[stage-18]

## Motivation and Context
Playlist UI test coverage.

## How Has This Been Tested?
Tests pass; no functionality changes.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: test/unit/template-editor/components/directives/dtv-component-playlist.spec.js [hunk 1]
```diff
@@ -436,6 +547,119 @@ describe("directive: templateComponentPlaylist", function() {
...
+        ]
+      };
+
+      $scope.getAvailableAttributeData = function(componentId, attributeName) {
+        return copySampleAttributeData[attributeName];
+      };
+
+      directive.show();
+
+      $scope.addItemJson(0)
```
Comment (ezequielc) [inline test/unit/template-editor/components/directives/dtv-component-playlist.spec.js:569]: @alex-deaconu Could you clarify why you need to pass `0` here? Or is it a mistake? Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good 👍 
Only a minor clarification question. Thanks!

File: test/unit/template-editor/components/directives/dtv-component-playlist.spec.js (hunk 1 above)
Comment (alex-deaconu) [reply test/unit/template-editor/components/directives/dtv-component-playlist.spec.js]: @ezequielc good catch, copy and paste error from the test suite below. Fixed, thanks!

Comment (ezequielc) [reply test/unit/template-editor/components/directives/dtv-component-playlist.spec.js]: @alex-deaconu 👍

**Pull request merged**